src/services/apiService.js

```diff
@@ -0,0 +1,73 @@
+export const apiService = {
+  BASE_API_URL: 'https://blue-bottle-api-test.herokuapp.com',
+  GET_COFFEE_SHOPS: '/v1/coffee_shops',
+  GET_TOKEN: '/v1/tokens',
+
+  async getNearestShops(position) {
+    const coffeeShops = await this.getCoffeeShops();
+    return coffeeShops.map(coffeeShop => {
+      return {
+        id: coffeeShop.id,
+        name: coffeeShop.name,
+        latitude: parseFloat(coffeeShop.x),
+        longitude: parseFloat(coffeeShop.y)
+      }
+    });
+  },
+
+  async getCoffeeShops(apiKey = '') {
```

Comment:
Argument values should not be reassigned, because then it becomes unpredictable what values do they have